samples/base-qcow2.json

```diff
@@ -67,8 +67,9 @@
     {
       "name": "org.osbuild.qcow2",
       "options": {
-	"filename": "base.qcow2",
-        "root_fs_uuid": "76a22bf4-f153-4541-b6c7-0332c0dfaeac"
+        "filename": "base.qcow2",
+        "root_fs_uuid": "76a22bf4-f153-4541-b6c7-0332c0dfaeac",
+        "size": 3221225472
```

Comment:
The same fix-up is needed for the test case in `test/pipelines`.

Reply:
Maybe I should think after rebasing :smile: Thanks!